src/NuGet/Microsoft.Net.Compilers.netcore.nuspec

```diff
@@ -0,0 +1,61 @@
+                <dependency id="System.AppContext" version="4.0.1-beta-23401" />
+                <dependency id="System.Collections" version="4.0.11-beta-23401" />
+                <dependency id="System.Collections.Immutable" version="1.1.36" />
+                <dependency id="System.Console" version="4.0.0-beta-23401" />
+                <dependency id="System.Diagnostics.Debug" version="4.0.11-beta-23401" />
+                <dependency id="System.Diagnostics.FileVersionInfo" version="4.0.0-beta-23401" />
+                <dependency id="System.Diagnostics.Process" version="4.1.0-beta-23401" />
+                <dependency id="System.Diagnostics.Tools" version="4.0.1-beta-23401" />
+                <dependency id="System.Dynamic.Runtime" version="4.0.11-beta-23401" />
+                <dependency id="System.IO.FileSystem" version="4.0.1-beta-23401" />
+                <dependency id="System.IO.Pipes" version="4.0.0-beta-23401" />
+                <dependency id="System.Linq" version="4.0.1-beta-23401" />
+                <dependency id="System.Private.Uri" version="4.0.1-beta-23401" />
+                <dependency id="System.Reflection" version="4.1.0-beta-23401" />
+                <dependency id="System.Reflection.Metadata" version="1.1.0-alpha-00014" />
+                <dependency id="System.Reflection.Primitives" version="4.0.1-beta-23401" />
+                <dependency id="System.Resources.ResourceManager" version="4.0.1-beta-23401" />
+                <dependency id="System.Runtime" version="4.0.21-beta-23401" />
+                <dependency id="System.Runtime.Extensions" version="4.0.11-beta-23401" />
+                <dependency id="System.Runtime.Handles" version="4.0.1-beta-23401" />
+                <dependency id="System.Runtime.InteropServices" version="4.0.21-beta-23401" />
+                <dependency id="System.Runtime.Loader" version="4.0.0-beta-23401" />
+                <dependency id="System.Runtime.Serialization.Json" version="4.0.1-beta-23401" />
+                <dependency id="System.Security.Cryptography.Algorithms" version="4.0.0-beta-23401" />
+                <dependency id="System.Text.Encoding" version="4.0.11-beta-23401" />
+                <dependency id="System.Text.Encoding.CodePages" version="4.0.1-beta-23401" />
+                <dependency id="System.Text.Encoding.Extensions" version="4.0.11-beta-23401" />
+                <dependency id="System.Threading" version="4.0.11-beta-23401" />
+                <dependency id="System.Threading.Tasks" version="4.0.11-beta-23401" />
+                <dependency id="System.Threading.Tasks.Parallel" version="4.0.1-beta-23401" />
+                <dependency id="System.Threading.Thread" version="4.0.0-beta-23401" />
+                <dependency id="System.Xml.XDocument" version="4.0.11-beta-23401" />
+                <dependency id="System.Xml.XmlDocument" version="4.0.1-beta-23401" />
+            </group>
```

Comment:
Actually, hmm, is this what we want?  This means that if you reference this nuget, you get references to all of these.  What we really need here is development time dependencies, so these are only used for running the compiler, not referenced by the app.

Tagging @davidfowl, @yishaigalatzer, @ericstj